lib/main_common.pm

```diff
@@ -2411,6 +2411,12 @@ sub load_security_tests_tpm2 {
     }
 }
 
+sub load_security_tests_openssl {
+    load_security_console_prepare;
+
+    loadtest "security/openssl/TLSv1_3";
```

Comment:
I think the test case name should be changed to lower-case letters. I did not find any document to mention it, but as I know all the *.pm file name are lower-case letters.

Reply:
Thanks Ben,
I will modify it

Reply:
Done